tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -86,11 +85,11 @@ public function testFixProvider()
                 '<?php function xyz($a=10,         $b=20 , $c=30) {}',
                 '<?php function xyz($a=10,         $b=20, $c=30) {}',
             ),
-            'test method call' => array(
+            'test method call (I)' => array(
```

Comment:
add these to key to prevent having duplicated keys (which effectively causes some tests to be not run)